controllers/chat/sendSignedMessage.js

```diff
@@ -86,6 +87,8 @@ const sendSignedMesage = async (req, res) => {
     if (createdChannel?.channel?.is_channel_blocked)
       return res.status(403).json(responseError('Channel is blocked'));
 
+    let channelMember = [];
+    let senderInfo = [];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These variables are declared but not initialized with any values. It's better to initialize them with default values to avoid potential issues with `undefined` values later on.
```diff
- let channelMember = [];
- let senderInfo = [];
+ let channelMember = null;
+ let senderInfo = null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->